Cargo.toml

```diff
@@ -19,24 +19,11 @@ opt-level = "z"
 name = "eszip"
 path = "src/lib.rs"
 
-[[example]]
-name = "eszip_builder"
-path = "src/examples/builder.rs"
-
-[[example]]
-name = "eszip_viewer"
-path = "src/examples/viewer.rs"
-
-[[example]]
-name = "eszip_load"
-path = "src/examples/load.rs"
```

Comment:
Removed because of circular dependency between eszip and deno_core (was getting an error upgrading a dependency).  Opened https://github.com/denoland/eszip/issues/64

Reply:
Yikes, how can we alleviate it?

Reply:
I don't know 😅

Reply:
Why does `deno_core` have a dep on eszip?

Reply:
I think it's not `deno_core`, but `deno` (for `deno compile`)

Reply:
But then why is that an issue? The examples only rely on `deno_core`, not `deno`.

Reply:
As discussed on discord, it was the `once_cell` dep being pinned.